modules/custom-status/rest/custom-status-endpoint.php

```diff
@@ -339,17 +365,43 @@ public static function handle_reorder_status( WP_REST_Request $request ) {
 
 	// Utility functions
 
-	private static function is_valid_user_ids( $user_ids ) {
-		foreach ( $user_ids as $user_id ) {
-			$user = get_user_by( 'id', $user_id );
-			if ( ! $user instanceof WP_User ) {
+	private static function is_valid_editorial_metadata_ids( $metadata_ids ) {
+		// If the array is empty, it's valid
+		if ( [] === $metadata_ids ) {
+			return true;
+		}
+
+		// ToDo: Optimize this to batch fetch metadata terms, and only include the ID field.
+		foreach ( $metadata_ids as $metadata_id ) {
```

Comment:
Same thing as below needs to happen here, I've punted it for later as that's a new method that needs to go into EM.